packages/app/src/cli/services/context.ts

```diff
@@ -161,31 +173,44 @@ async function ensureIncludeConfigOnDeploy({
     includeConfigOnDeploy: previousIncludeConfigOnDeploy,
   })
 
-  if (force || previousIncludeConfigOnDeploy !== undefined) return
+  if (force || previousIncludeConfigOnDeploy === true) return
   await promptIncludeConfigOnDeploy({
     appDirectory: app.directory,
     localApp: app,
   })
 }
 
+async function removeIncludeConfigOnDeployField(localApp: AppInterface) {
+  const patch = {build: {include_config_on_deploy: undefined}}
+  await patchAppConfigurationFile({path: localApp.configuration.path, patch, schema: localApp.configSchema})
+  const message = {
+    headline: `Configuration is now included on deploy`,
+    body: [
+      `The \`include_config_on_deploy\` field has been removed from your configuration file and is now enabled by default.`,
```

Comment:
```suggestion
      `The \`include_config_on_deploy\` field is no longer supported, since all apps must now include configuration on deploy. It has been removed from your configuration file.`,
```